packages/flow-core/src/components/f-file-upload/f-file-upload.ts

```diff
@@ -484,15 +484,15 @@ export class FFileUpload extends FRoot {
 										<f-text variant="para" size="small" weight="regular" state="secondary"
 											>${this.fileType === "all"
 												? `(All formats supported)`
-												: `(${this.fileType})`}</f-text
+												: `(${getExtensionsFromMimeType(this.fileType ?? "")})`}</f-text
```

Comment:
Since you are passing the blank string to `getExtensionsFromMimeType` can you handle that condition in the function itself so that it won't do that processing?